### PR TITLE
only implement Meta::thriftIDL when the thriftFileName is provided

### PR DESCRIPTION
**Pull request by ShanniLi**
r @Raynos @kriskowal 

ref: #46

### CODE REVIEW

File: hyperbahn-client.js [hunk 1]
```diff
@@ -188,6 +188,7 @@ function getThriftSync(options) {
 
     return channel.TChannelAsThrift({
         source: thriftSource,
+        thriftFileName: options.thriftFile.split("/").pop(),
```
Comment (kriskowal) [inline hyperbahn-client.js]: https://nodejs.org/api/path.html#path_path_basename_p_ext

**Comment by kriskowal** [comment]
@Raynos I find this change strictly less useful and will delay useful outcomes for tcurl.

**Comment by ShanniLi** [comment]
+1 for @kriskowal

**Comment by kriskowal** [comment]
:ok:

**Pull request merged**